api/tests/mocha/controllers/changes.js

```diff
@@ -32,6 +35,7 @@ describe('Changes controller', () => {
   });
 
   beforeEach(() => {
+    realSetTimeout = setTimeout;
```

Comment:
Why not just inline `setTimeout`?

Reply:
hm? I need to save the proper `setTimeout` function (which gets overriden by `useFakeTimers` on next line) to use as `nextTick` (the promise chain became unusable). 
Do you have another suggestion?

Reply:
Nevermind ... it's too late and I can't think straight :) Thanks!

Reply:
Ahh, it's overridden. That makes sense.